display_information/show-callout/README.md

```diff
@@ -19,7 +19,7 @@ Click anywhere on the map. A callout showing the WGS84 coordinates for the click
 3. Get the screen point's location on the map using `MapView.screenToLocation(Point2D)`.
 4. Get the `MapView`'s callout.
 5. Set the title and detail of the callout to display the map point's coordinates.
-6. Display the callout at the map point and specify an animation duration by using `callout.showCalloutAt(mapPoint, duration)`.
+6. Display the callout at the map point `callout.showCalloutAt(mapPoint)`.
```

Comment:
@jenmerritt I think we should definitely raise an issue for the callout animation. Not sure how to describe it, but it's almost as if the animation duration is 'stacking' if we initiate another animation before it's finished. Feel free to log an issue (ideally add a snippet of the sample code and a gif), or give me a shout if there's anything to clarify.